etc/ex.dev.config.yaml

```diff
@@ -17,4 +17,7 @@ openshift:
   target: localhost:8443
   user: OPENSHIFT_USER
   pass: OPENSHIFT_PASS
-devbroker: true
+broker:
+  devbroker: true
+  launchapbonbind: false
```

Comment:
There are more config files than the dev one.  Personally, I think we should only have one config file for the broker. Let's go with just this change and I'll follow up to remove the other config files and add a doc explaining the use cases. Created https://github.com/openshift/ansible-service-broker/issues/170.